Source/WebGPU/WGSL/Metal/MetalFunctionWriter.cpp

```diff
@@ -91,8 +91,9 @@ class FunctionDefinitionWriter : public AST::Visitor {
     void visit(AST::Parameter&) override;
     void visitArgumentBufferParameter(AST::Parameter&);
 
-private:
     StringBuilder& m_stringBuilder;
```

Comment:
Unless it's against the WebKit coding standard, having public members starts with `m_` is a bit on the nose. Can we make it `StringBuilder& stringBuilder`?

Reply:
I don't think we should mix `m_` and non-`m_` fields, although it's not written anywhere, but I also don't think we should mix public and private `m_` fields. Usually fields are all public, without the `m_` prefix, or all private with the prefix. I'll just revert this change and add an accessor method.